### PR TITLE
F/T sensor fix and cable collisions

### PR DESCRIPTION
**Pull request by VladimirIvan**
- Adds collision spheres to account for dangling cables around the wrist.
- Fixes F/T sensor frames
- Adds F/T sensor gazebo tags
- Updates nextagea_moveit_config to account for new links in the collision matrix

### CODE REVIEW

File: nextagea_description/urdf/NextageA_robot.xacro [hunk 1]
```diff
@@ -689,4 +689,48 @@
     <implicitSpringDamper>1</implicitSpringDamper>
   </gazebo>
 
+  <joint name="LARM_JOINT4_Link_cables_joint" type="fixed">
+        <parent link="LARM_JOINT4_Link"/>
+        <child link="LARM_JOINT4_Link_cables"/>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <axis xyz="0 0 1"/>
+    </joint>
```
Comment (christian-rauch) [inline nextagea_description/urdf/NextageA_robot.xacro:697]: The indentation of the `joint` tag start/end is not aligned.

File: nextagea_moveit_config/.setup_assistant [hunk 2]
```diff
@@ -1,8 +1,8 @@
 moveit_setup_assistant_config:
   URDF:
     package: nextagea_description
-    relative_path: urdf/NextageAOpen.urdf.xacro
-    xacro_args: ""
+    relative_path: urdf/NextageAOpen_gazebo_intel_realsense.urdf.xacro
```
Comment (christian-rauch) [inline nextagea_moveit_config/.setup_assistant:4]: Do we have a generic xacro without the sensor?

File: nextagea_description/urdf/NextageA_robot.xacro [hunk 3]
```diff
@@ -689,4 +689,48 @@
...
+            <xacro:nxa_box_inertia_def x="0.016" y="0.006" z="0.012" mass="0.01"/>
+        </inertial>
+
+        <collision>
+            <origin xyz="0 -0.08 -0.1" rpy="0 0 0"/>
+            <geometry>
+                <sphere radius="0.1"/>
+            </geometry>
+        </collision>
+    </link>
```
Comment (christian-rauch) [inline nextagea_description/urdf/NextageA_robot.xacro:734]: Not so important, but if we get more of these symmetric collision links we should make this a macro that is called with an 'L' or 'R' prefix, like here:
https://github.com/ipab-slmc/nextagea/blob/2e704c60146c4aba03fefaf1fb5920a70aaf296b/nextagea_description/urdf/NextageAOpen.urdf.xacro#L62-L67

**Review by christian-rauch — CHANGES_REQUESTED** [review]

File: nextagea_moveit_config/.setup_assistant (hunk 2 above)
Comment (VladimirIvan) [reply nextagea_moveit_config/.setup_assistant:4]: We do but the way moveit works, you can generate the SRDF in setup assistant with the camera links, include them in the collision matrix, and if they are not present in the simpler model, moveit will just print a warning. If you use the simpler model to generate the SRDF instead and then load the one with the camera links, the camera link automatically collide and may cause problems.

File: nextagea_description/urdf/NextageA_robot.xacro (hunk 3 above)
Comment (VladimirIvan) [reply nextagea_description/urdf/NextageA_robot.xacro:734]: Definitely, I don't think we'll need more but if do, we can put them in a macro.

**Review by christian-rauch — APPROVED** [review]

**Comment by christian-rauch** [comment]
Argh, should have tested this on the robot.

I am getting the error message:
```
[ERROR] [1620399925.906313841]: link 'LARM_JOINT4_Link_cables' is not unique.
[ERROR] [1620399925.973866310]: Unable to parse URDF
```
probably due to some copy&paste error @VladimirIvan .

**Comment by VladimirIvan** [comment]
I moved the cable links from the urdf in `nextage_dynamic_pick_place` into here. Since that urdf extends this one it now has duplicate links. We have to remove these links from that repo and this error will go away.

**Pull request merged**